src/Test.UnitTests.Sarif.Multitool/MergeCommandTests.cs

```diff
@@ -34,7 +34,7 @@ public class MergeCommandTestsFixture : DeletesOutputsDirectoryOnClassInitializa
         [Trait(TestTraits.Bug, "https://github.com/microsoft/sarif-sdk/issues/1592")]
         public void MergeCommand_WhenThereAreNoInputFiles_ProducesEmptyRunsArray()
         {
-            RunTest("NoInputFiles.sarif");
+//            RunTest("NoInputFiles.sarif");
```

Comment:
>NoInputFiles [](start = 23, length = 12)

Eddy, we need to uncomment this, probably a test failure to resolve here.

Reply:
Fixed!